en/components/general/ignite-ui-licensing.md

```diff
@@ -197,3 +217,16 @@ The best way to define an environment variable depends on what type of informati
 
 * encrypt it and add it [to your .travis.yml](https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml)
 * add it to your [Repository Settings](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings)
+
+### GitHub Actions Configuration
+Azure Pipelines and GitHub Actions both allow you to [create workflows](https://help.github.com/en/actions/migrating-to-github-actions/migrating-from-azure-pipelines-to-github-actions#migrating-script-steps) that automatically build, test, publish, release, and deploy code.
```

Comment:
What's Azure pipelines have to do with all this? And the migration link - no getting started link available?